TCPEchoServer/Socket.cpp

```diff
@@ -0,0 +1,95 @@
+#include "pch.h"
+#define WIN32_LEAN_AND_MEAN
+#define _WINSOCK_DEPRECATED_NO_WARNINGS
+#include <ws2tcpip.h>
+#include "Socket.h"
+
+Socket::Socket()
+{
+	m_s = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
```

Comment:
Your constructor should duplicate the 'socket' parameters

Reply:
Done